src/main/java/lotto/domain/LottoBundle.java

```diff
@@ -0,0 +1,34 @@
+package lotto.domain;
+
+import java.util.EnumMap;
+import java.util.List;
+
+public class LottoBundle {
+
+    private final List<Lotto> lottoBundle;
+
+    public LottoBundle(List<Lotto> lottoBundle) {
+        this.lottoBundle = lottoBundle;
+    }
+
+    public EnumMap<Rank, Integer> makeStatistics(WinningNumbers winningNumbers) {
+
+        EnumMap<Rank, Integer> rankIntegerEnumMap = Rank.makeDefaultMap();
+
+        for (Lotto lotto : lottoBundle) {
+            Rank currentRank = Rank.checkPrize(winningNumbers.checkMatchCount(lotto),
+                    winningNumbers.checkMatchBonus(lotto));
+            rankIntegerEnumMap.put(currentRank, rankIntegerEnumMap.get(currentRank) + 1);
+        }
+
+        return rankIntegerEnumMap;
+    }
```

Comment:
Rank 객체를 생성하는 부분은 WinningNumbers에게 위임할 수 있지 않을까요? 
`WinningNumbers` 는 `Lotto` 하나를 받아 `Rank`를 반환한다.

특별한 이유가 없으면 EnumMap 보다는 Map을 사용해 사용하는 쪽에서는 인터페이스에 의존하는 것이 좋습니다. 다만 구현체는 EnumMap을 사용하면 될 것 같아요. 

```suggestion
    public Map<Rank, Integer> makeStatistics(WinningNumbers winningNumbers) {

        Map<Rank, Integer> rankIntegerEnumMap = Rank.makeDefaultMap();

        for (Lotto lotto : lottoBundle) {
            Rank currentRank = Rank.checkPrize(winningNumbers.checkMatchCount(lotto),
                    winningNumbers.checkMatchBonus(lotto));
            rankIntegerEnumMap.put(currentRank, rankIntegerEnumMap.get(currentRank) + 1);
        }

        return rankIntegerEnumMap;
    }
```